draft-packages/badge/KaizenDraft/Badge/decision-tokens.scss

```diff
@@ -0,0 +1,5 @@
+@import "~@kaizen/design-tokens/sass/color";
+
+// if --kz-var-color-ash fails, assume a Zen theme and fall back to $kz-color-wisteria-200
+// (the non-css variable token)
+$dt-color-background-default: var(--kz-var-color-ash, $kz-color-wisteria-200);
```

Comment:
@jasononeil keen to know your thoughts, but i think it might be better to use these modules:
`@import "~@kaizen/design-tokens/sass/color-vars";`
`@import "~@kaizen/design-tokens/sass/variable-identifiers";`

and change it to:
`$dt-color-background-default: var($kz-var-id-color-ash, $kz-var-color-wisteria-200);`

That way, the CSS variable identifier `--kz-var-color-ash`, will be used in a safer way (compiler will yell at you).
And, `$kz-color-wisteria-200` becomes a CSS variable, and we further remove concrete values.

Reply:
I didn't know that existed! Thanks